core/controllers/topics_and_skills_dashboard.py

```diff
@@ -149,10 +150,22 @@ class NewSkillHandler(base.BaseHandler):
     def post(self):
         description = self.payload.get('description')
         linked_topic_ids = self.payload.get('linked_topic_ids')
+        explanation = self.payload.get('explanation')
         rubrics = self.payload.get('rubrics')
         if not isinstance(rubrics, list):
             raise self.InvalidInputException('Rubrics should be a list.')
 
+        if not isinstance(explanation, dict):
+            raise self.InvalidInputException(
+                'Review material should be an object.')
```

Comment:
Wouldn't "Explanation should be a dict." be more appropriate here (trying to use the standard terminology)?

Perhaps better to call the argument explanation_dict, too (to differentiate it from a domain object).

Reply:
Done